modules/sc-mesh-secure-deployment/src/nats/src/cbma_adaptation.py

```diff
@@ -150,9 +146,7 @@ def __create_vlan_interfaces(self) -> None:
                     )
 
                     # Bring VLAN interface up
-                    subprocess.run(
-                        ["ip", "link", "set", vlan_name, "up"], check=True
-                    )
+                    subprocess.run(["ip", "link", "set", vlan_name, "up"], check=True)
```

Comment:
I think this might make the PEP linter tool unhappy :cake:

Reply:
Will check

Reply:
I don't follow. Or are you referring to line length? I've been using black tool with default setting that limits line length to 88 instead of 79 that is what flake8 uses by default. Pylint itself is not complaining.

Reply:
I was referring to the line length, I thought you were using 79, no problem then :pie: